.gitmodules

```diff
@@ -1,6 +1,3 @@
 [submodule "azure-c-shared-utility"]
 	path = azure-c-shared-utility
 	url = https://github.com/Azure/azure-c-shared-utility.git
-[submodule "libwebsockets"]
-	path = libwebsockets
-	url = https://github.com/warmcat/libwebsockets.git
```

Comment:
And another test comment

---
In reply to: [74841354](https://github.com/Azure/azure-uamqp-c/pull/15#discussion_r74841354) [](ancestors = 74841354)